nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,17 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 2.6.1
+***************
+
+Sockets
+=======
+
+* Fixed a bug where the :c:func:`nrf_recv` and :c:func:`nrf_recvfrom` functions erroneously returned ``-1`` and set ``errno`` to :c:macro:`NRF_EAGAIN` instead of returning ``0`` when these three conditions were met:
+  * :c:func:`nrf_recv` and :c:func:`nrf_recvfrom` was called with flag :c:macro:`NRF_MSG_DONTWAIT`
+  * socket was closed by the server
+  * there were no more data to read (end of file)
```

Comment:
```suggestion
* Fixed a bug where the :c:func:`nrf_recv` and :c:func:`nrf_recvfrom` functions erroneously returned ``-1`` and set ``errno`` to :c:macro:`NRF_EAGAIN` instead of returning ``0`` when these three conditions were met:

  * The :c:func:`nrf_recv` and :c:func:`nrf_recvfrom` functions were called with a flag :c:macro:`NRF_MSG_DONTWAIT`.
  * The socket was closed by the server.
  * There was no more data to read (End Of File (EOF)).
```